SmartDeviceLinkTests/ProxySpecs/SDLSecondaryTransportManagerSpec.m

```diff
@@ -491,44 +496,67 @@ + (void)swapConnectionMethods {
                 testPrimaryProtocol = [[SDLProtocol alloc] init];
                 testPrimaryTransport = [[SDLTCPTransport alloc] init];
                 testPrimaryProtocol.transport = testPrimaryTransport;
+
                 dispatch_sync(testStateMachineQueue, ^{
                     [manager startWithPrimaryProtocol:testPrimaryProtocol];
                 });
 
                 manager.secondaryTransportType = SDLTransportSelectionIAP;
             });
 
-            it(@"should transition to Connecting state", ^{
+            it(@"should transition to Connecting state but before RAIR is sent", ^{
```

Comment:
I'm not understanding what this means. What is this testing? It's testing that the security manager is `nil` here?